assignment-2-ruby/rborder/order_items_data.rb

```diff
@@ -0,0 +1,56 @@
+AISLES = ['Produce','Dairy & Eggs','Meat & Seafood','Snacks','Deli','Frozen Foods','Bulk','Household','Bakery']
+
+ITEMS = [{name: "Lettuce",  aisle: "Produce",   cost: 1},
```

Comment:
could look into creating and ingesting a csv/yaml file for this type of structured data

Reply:
I think YAML files are coming up, but I didn't want to jump too far ahead.